### DISCUSSION TITLE
读书这件事

### DISCUSSION THREAD

**Discussion by giscus** [edony-ink/comments#43] [General]
# 读书这件事

读书、听书、短视频等等都可以算是读书，只是读书这件事就好比吃饭，有的是别人帮你嚼碎了再喂给你、有的是自己动手烹饪和享用、有的是快餐速食填饱肚子，营养可能还是那些，只是味道因人而异。

https://www.edony.ink/about-reading/

**Comment by 0xZhangKe** [comment]
一边写代码一边听小说挺舒服的，尤其是面对的是屎山。

**Reply by edonyzpc** [reply]
非常有意思的做法，下次我去试一下。
我现在面对屎山代码的时候一般都是听 rapper，如果实在太屎山一般都会自己唱一句“我是恁爹”……
